.travis.yml

```diff
@@ -59,4 +59,12 @@ install: travis_retry composer install --no-interaction --prefer-source
 
 script:
   - composer phpcs
-  - composer tests
+  - composer tests-travis
+
+after_script:
+  - wget https://scrutinizer-ci.com/ocular.phar
+  - if [ $(phpenv version-name) = "5.3" ] && [ "$REMOVE_XDEBUG" = "0" ]; then php ocular.phar code-coverage:upload --format=php-clover test/clover.xml --revision=$TRAVIS_COMMIT; fi
```

Comment:
It **must** be tested on branch 2.0. But **must not** be testes on the branch where is failure available.